src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -144,20 +200,32 @@
  (fn [{:keys [db]} [{:keys [message]}]]
    (let [{:keys [offset ownershipStatus collectibles
                  hasMore]} (transforms/json->clj message)
+         ownershipStatus   (update-keys ownershipStatus name)
+         owner-address     (some-> ownershipStatus
+                                   first
+                                   key)
+         ownership-status  (-> (get ownershipStatus owner-address)
+                               (update-keys (comp js/parseInt name)))
          collectibles      (cske/transform-keys transforms/->kebab-case-keyword collectibles)
          pending-requests  (dec (get-in db [:wallet :ui :collectibles :pending-requests]))
-         owner-address     (some->> ownershipStatus
-                                    first
-                                    key
-                                    name)]
+         ;; collectibles are never fetched and cached for this address before
+         new-account?      (->> ownership-status
+                                vals
+                                (every?
+                                 #(and (= (:state %) (ownership-state :updating))
+                                       (= (:timestamp %) -1))))]
```

Comment:
Maybe it's better to move this to a named function so that we avoid using `#()` in a non-one-liner function and we give a name to this predicate.